CSharp/Library/Microsoft.Bot.Connector.Shared/JwtConfig.cs

```diff
@@ -9,9 +9,12 @@
     public static class JwtConfig
     {
         /// <summary>
-        /// TO CHANNEL FROM BOT: Login URL
+        /// TO CHANNEL FROM BOT: Login URL format string. Bot developer can optionally specify

```

Comment:
The comment should also say "URL Template", rather than "Format String".

Reply:
will fix.